docs/ktools.md

```diff
@@ -366,3 +366,21 @@ This is the line of `definition.kore` that the axiom appears on as
 well as the original location of the rule in the K semantics. You can
 use this information to figure out which rules and functions are
 causing the most time and optimize them to be more efficient.
+
+Running tests - kserver
+--------------------------
+
+The `kserver` is a front-end tool based on [Nailgun](https://github.com/facebookarchive/nailgun)
+and it helps a lot with reducing the startup time of the JVM. Calling `kserver` in a terminal
+window will wait for all kompile/kprove calls and force them to run in the same process,
+and share the same threads. This also reduces significantly the thread contention. `kompile`
```

Comment:
"reduces signficantly the thread contention" -> "reduces the thread contention significantly"